### PR TITLE
Css/loader

### PR DESCRIPTION
**Pull request by davodey**

### CODE REVIEW

File: progress-bar/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Pearson Datepicker Web Component
+# Pearson Progress Bar Web Component
```
Comment (dengeist) [inline progress-bar/README.md]: I don't think this commit is supposed to be here :P

File: tab-navigation/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Pearson Datepicker Web Component
+# Pearson Tab Navigation Web Component
```
Comment (dengeist) [inline tab-navigation/README.md]: Orrrr this one :P

**Comment by davodey** [comment]
@dengeist I actually just fixed the nesting and moved the selectors with ',' to new lines - the styles were never moved, it just looks that way.

**Comment by davodey** [comment]
@dengeist im going to leave those commits in.  I needed to change the readmes and forgot to stash my files - it needs to be done regardless :)

**Pull request merged**